### ISSUE TITLE
Where did Doc.from_orth go?

### ISSUE THREAD

**Issue by mdda** [spacy-io/spaCy#392]
In spacy/tokens/doc.pyx :
```
Doc.__init__(self, Vocab vocab, orths_and_spaces=None):
```
orths_and_spaces doesn't appear to do anything.  

I'm puzzled, since https://spacy.io/docs/tutorials/byo-annotations says that ```tokens_from_list()``` is deprecated in favour of the ```list( (token, has_space), ... )``` version.

(btw :  spaCy looks wonderful!  I'm just having a few teething troubles...)

**Comment by honnibal** [comment]
This slipped between the cracks while I was refactoring. Sorry!

**Comment by mdda** [comment]
(sorry to ask a continuation question - but you replied to the issue so quickly....)

If I construct a Doc explicitly (my own tokenization, probably the same as spaCy, but can't absolutely guarantee it), I notice that doing :

```
spacy_nlp.tagger(doc)
spacy_nlp.entity(doc)
```
finds more entities than if I just call ```.entity(doc)```.  Presumably, the NER uses the tags as NER boundary hints, etc.

Would NER performance be improved (or changed in any way) if I also include ```spacy_nlp.parser(doc)``` in the pipeline?  i.e.  Does the NER make use of dependency information?

**Comment by honnibal** [comment]
The current NER model doesn't use the dependency parse, no. I've always
meant to try that, but never got around to it.

**Comment by honnibal** [comment]
The `orths_and_spaces` argument is now fixed, and the fix will be released in 1.0.
